Marlin/src/pins/pins_MKS_SBASE.h

```diff
@@ -104,14 +106,16 @@
 
 //
 // Connector J7
+// Note: These pins are all shared with the EXP1/EXP2 Connector.
+// Using them with an LCD connected or configured will lead to hangs & crashes.
 //
 
 // 5V
 // NC
 // GND
 #define PIN_P0_17          P0_17
 #define PIN_P0_16          P0_16
-#define PIN_P0_14          P0_14
+#define PIN_P0_14          P0_15
```

Comment:
This definition seems a little strange. Should the define now read PIN_P0_15?

Reply:
good catch